src/views/Reports/index.tsx

```diff
@@ -289,7 +290,16 @@ function Reports(props: ReportsProps) {
                         isSignedOff: item?.lastGeneration?.isSignedOff,
                     }),
                 ),
-                actionColumn,
+                createActionColumn<ReportFields, string>(
+                    'action',
+                    '',
+                    (item) => ({
+                        id: item?.id,
```

Comment:
I think item will be defined.
```suggestion
                        id: item.id,
```
Can you also make this change to the createStatusColumn?